clash-protocols/clash-protocols.cabal

```diff
@@ -77,6 +77,7 @@ common common-options
     -fplugin GHC.TypeLits.Extra.Solver
     -fplugin GHC.TypeLits.Normalise
     -fplugin GHC.TypeLits.KnownNat.Solver
+    -fconstraint-solver-iterations=6
```

Comment:
I would rather move this to the offending module(s). This is a bit of a sledgehammer.